src/client/runner/automation/playback/press.js

```diff
@@ -1,38 +1,151 @@
+import hammerhead from '../../deps/hammerhead';
 import testCafeCore from '../../deps/testcafe-core';
-import testCafeUI from '../../deps/testcafe-ui';
-import * as automationUtil from '../util';
-import keyPressSimulator from './key-press-simulator';
-import async from '../../deps/async';
+import KeyPressSimulator from '../key-press-simulator';
+import supportedShortcutHandlers from '../shortcuts';
+import each from '../../utils/promise-each';
+import wait from '../../utils/promise-wait';
+import { ACTION_STEP_DELAY } from '../settings';
 
-var SETTINGS     = testCafeCore.SETTINGS;
+var Promise      = hammerhead.Promise;
+var browserUtils = hammerhead.utils.browser;
+
+var arrayUtils   = testCafeCore.arrayUtils;
 var domUtils     = testCafeCore.domUtils;
 var keyCharUtils = testCafeCore.keyCharUtils;
 
-var selectElement = testCafeUI.selectElement;
 
+const KEY_PRESS_DELAY = 80;
+
+
+export default class PressAutomation {
+    constructor (keysString) {
+        this.keyCombinations = keyCharUtils.parseKeysString(keysString).combinations;
+        // NOTE: preventing of 'keydown' and 'keypress' events for select element
```

Comment:
I can't understand why this comment is here. Is it related to the `this.forceShortcut`? If so it doesn't tell me anything. Maybe we should move it anywhere?